test/test_shapes.c

```diff
@@ -110,6 +110,14 @@ int main()
             par_shapes_export(m, "build/test_shapes_rock.obj");
             par_shapes_free_mesh(m);
 
+            m = par_shapes_create_cone(15, 3);
+            par_shapes_export(m, "build/test_shapes_cone.obj");
+            par_shapes_free_mesh(m);
+
+            m = par_shapes_create_parametric_disk(15, 3);
+            par_shapes_export(m, "build/test_shapes_parametric_disk.obj");
+            par_shapes_free_mesh(m);
```

Comment:
Here's a screenshot of the texture coordinates and normals of the `test_shapes_parametric_disk.obj`. The wireframe of `test_shapes_cone.obj` is shown for reference

![image](https://user-images.githubusercontent.com/1313794/102919736-4ab86300-4481-11eb-9c67-779ab00d4009.png)